src/contour/ui.template/Terminal.qml.in

```diff
@@ -28,6 +28,31 @@ ContourTerminal
         focus : false
     }
 
+
+
+    Rectangle {
+        Timer {
+            id: sizeWidgetTimer
+            interval: 1000;
+            running: false;
+            onTriggered: sizeWidget.visible = false
+        }
+        id : sizeWidget
+        anchors.centerIn: parent
+        width:  80
+        height: 30
+        color: "white"
+        visible : false
+        focus : false
+        Text {
+            id : sizeWidgetText
+            anchors.centerIn: parent
+            font.pointSize: 12
+            text :  session.displaytColumnsCount.toString() + "x" + session.displayLinesCount.toString()
```

Comment:
There was already at least `pageLineCount` read property, which reads from `terminal.pageSize()` rather than profile setting. Why didn't you pick this? The `pageLineCount` property also has notification support. We'd need to add a `pageColumnCount` companion though. I'm open in getting rid of `pageLineCount` in favor of `display{Foo}Count`, but there should probably be only one way (including notification access), to avoid confusion :)

Reply:
i changed it to `pageLineCount` and added `pageColumnsCount`